app/AppKernel.php

```diff
@@ -50,7 +50,7 @@ public function registerBundles()
             new Solution\MongoAggregationBundle\SolutionMongoAggregationBundle(),
         );
 
-        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
+        if (in_array($this->getEnvironment(), array('dev', 'test', 'behat'))) {
```

Comment:
Are debug tools required on the functionnal testing environnement or is it better to use those tests in a production-like environnement ?